src/encoded/types/access_key.py

```diff
@@ -18,4 +27,39 @@
     },)
 class AccessKey(SMAHTItem, SnovaultAccessKey):
     """AccessKey class."""
+    ACCESS_KEY_EXPIRATION_TIME = 90  # days
+    item_type = 'access_key'
+    name_key = 'access_key_id'
     schema = mixin_smaht_permission_types(SNOVAULT_ACCESS_KEY_SCHEMA)
+    embedded_list = []
+
+    STATUS_ACL = {
+        'current': [(Allow, 'role.owner', ['view', 'edit'])] + ONLY_ADMIN_VIEW_ACL,
```

Comment:
Did we talk about naming this last item something like `ELSE_ONLY_ADMIN_VIEW_ACL` or `OTHERWISE_ONLY_ADMIN_VIEW_ACL` to encourage it being last. It's in the right position here but it might be easier to spot it in the wrong position if it had a better name.